lms/static/js/verify_student/views/image_input_view.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Allow users to upload an image using a file input.
+ *
+ * This uses HTML Media Capture so that iOS will
+ * allow users to use their camera instead of choosing
+ * a file.
+ */
+
+ var edx = edx || {};
+
+ (function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.ImageInputView = Backbone.View.extend({
+
+        template: "#image_input-tpl",
+
+        initialize: function( obj ) {
+            this.$submitButton = obj.submitButton || "";
```

Comment:
The convention of using a leading $ for variables implies the variables is a DOM object. The line should be something like:
this.$submitButton - obj.submitButton? $(obj.submitButton) : '';